asciimatics/effects.py

```diff
@@ -685,7 +688,8 @@ def overlaps(self, other, use_new_pos=False):
             return True
 
     def _update(self, frame_no):
-        if frame_no % 2 == 0:
+        self._frame_no = frame_no
+        if self._speed == 0 or frame_no % self._speed == 0:
```

Comment:
Any chance you could update the tests (see https://github.com/peterbrittain/asciimatics/blob/5866bca2bdedb34596673f627249660910c7bf12/tests/test_effects.py#L376C1-L376C1), or add a new test to prove that it only updates on the specified speed?